libs/converter-nh-accuvote/src/convert/convert_election_definition_header.ts

```diff
@@ -82,6 +82,22 @@ export function convertElectionDefinitionHeader(
       paperSize = BallotPaperSize.Legal;
       break;
 
+    case '8.5X17':
+      paperSize = BallotPaperSize.Custom17;
+      break;
+
+    case '8.5X18':
+      paperSize = BallotPaperSize.Custom18;
```

Comment:
The data from the state says the ballot is 8.5X18, but it's not. We only warn on mismatch, but we do fail to parse the data if the ballot size isn't recognized at all.